sass/dimensions.scss

```diff
@@ -0,0 +1,5 @@
+$num-of-columns: 12;
```

Comment:
yeah we use a 24 col grid in composer :( when speaking to @jamespamplin @akemitakagi we agreed 12 was more standard so im going with that.  You can get a more fine-grained control by nesting eg:

``` html
<row>
  <column span="1">
    <row>
      <column span="6"></column>
      <column span="6"></column>
    </row>
  </column>
</row>
```

Now, I agree that this isn't the neatest. I can switch to 24 if needs be, but I guess we will see what happens when we integrate.